src/ApplicationRequest/Listener/SendConfirmationRequestSubscriber.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace AppBundle\ApplicationRequest\Listener;
+
+use AppBundle\ApplicationRequest\ApplicationRequestEvent;
+use AppBundle\ApplicationRequest\Events;
+use AppBundle\Mailer\MailerService;
+use AppBundle\Mailer\Message\ApplicationRequestConfirmationMessage;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
+
+class SendConfirmationRequestSubscriber implements EventSubscriberInterface
```

Comment:
What do you think to rename by `ApplicationRequestNotifier` ? 
1. It will be reused in the future easly for others notifications actions IMO. 
2. It will be consistent with the `AssessorRequestNotifier` :wink:

Reply:
as discussed in PM, I will not change the name `SendConfirmationRequestSubscriber` for keeping it clear and simple, as this listener is used only for sending a confirmation mail
thanks, @thomasbisignani for discussion ;)